diff --git a/src/bins/src/setup.rs b/src/bins/src/setup.rs
index b6a29a57..fd2fa0fd 100644
--- a/src/bins/src/setup.rs
+++ b/src/bins/src/setup.rs
@@ -60,6 +60,7 @@ fn main() -> Result<()> {
 fn main_inner() -> Result<()> {
     #[rustfmt::skip]
     let mut arg_config = Command::new("Setup")
+        .ignore_errors(true)
         .about(format!("Velopack Setup ({}) installs applications.\nhttps://velopack.io", env!("NGBV_VERSION")))
         .arg(arg!(-s --silent "Hides all dialogs and answers 'yes' to all prompts"))
         .arg(arg!(-v --verbose "Print debug messages to console"))
